tests/e2e/repo.test.e2e.js

```diff
@@ -0,0 +1,20 @@
+// @ts-check
+import {test, expect} from '@playwright/test';
+import {load_logged_in_context, login_user} from './utils_e2e.js';
+
+test.beforeAll(async ({browser}, workerInfo) => {
+  await login_user(browser, workerInfo, 'user2');
+});
+
+test('Create Repo', async ({browser}, workerInfo) => {
+  const context = await load_logged_in_context(browser, workerInfo, 'user2');
+  const page = await context.newPage();
+
+  const response = await page.goto('/repo/create');
+  await expect(response?.status()).toBe(200); // Status OK
```

Comment:
Can we add some visual assertions to make sure the new repo page elements are loading correctly? 

Maybe select some of (or all) of the dropdowns and make sure they appear? We've had issues with that breaking before.

Reply:
I wonder if it's reasonable to add ids to make the selectors a little less finicky

Reply:
Many of the dropdowns have selectors that collide which makes it a little tricky. Adding ids would be useful for more comprehensive testing.

Reply:
Yeah, I think we should add an ID where the selectors are more tricky.

Reply:
Should we tackle that in this PR?